app/src/main/AndroidManifest.xml

```diff
@@ -1,21 +1,24 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="co.infinum.complexify" >
+          package="co.infinum.complexify">
```

Comment:
Not important but, lets be consistent and go with com.bla.bla

Reply:
This is one of the questions I asked 😄 

> Should we rename the complexify-android module to just "complexify"? In fact, should we maybe move the repo to a different domain (infinum.com)?

The downside is that current users have to update their projects. The upside is that it's much nicer for the future

Reply:
They will welcome it probably and change is very minor.